### PR TITLE
Create dhlcgd.json

### PR DESCRIPTION
**Pull request by dhlcgd**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

image

### CODE REVIEW

File: domains/kitb2romjbz3.dhlcgd.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dhlcgd",
+        "email": "dhlcgdofficial@gmail.com"
+    },
+    "record": {
+        "CNAME": "gv-zz3nklni3izils.dv.googlehosted.com"
```
Comment (DEV-DIBSTER) [inline domains/kitb2romjbz3.dhlcgd.json:7]: Site not accessible.

File: domains/www.dhlcgd.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dhlcgd",
+        "email": "dhlcgdofficial@gmail.com"
+    },
+    "record": {
+        "CNAME": "ghs.google.com"
```
Comment (DEV-DIBSTER) [inline domains/www.dhlcgd.json:7]: Provide site preview.

File: domains/dhlcgd.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dhlcgd",
+        "email": "dhlcgdofficial@gmail.com"
+    },
+    "record": {
+        "CNAME": "ghs.google.com"
```
Comment (DEV-DIBSTER) [inline domains/dhlcgd.json:7]: Provide site preview.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/www.dhlcgd.json (hunk 2 above)
Comment (dhlcgd) [reply domains/www.dhlcgd.json:7]: My website is live at [dh-lcgd.blogspot.com](dh-lcgd.blogspot.com)

Comment (dhlcgd) [reply domains/www.dhlcgd.json:7]: To use Blogger, I need to point the CNAME to ghs.google.com and the other CNAME is used for Google to authenticate that I am allowed to use this domain. The site is currently not accessible via the new domain is-a.dev until the CNAME is added and Google allows me to update the new domain name to Blogger

File: domains/dhlcgd.json (hunk 3 above)
Comment (dhlcgd) [reply domains/dhlcgd.json:7]: My website is live at [dh-lcgd.blogspot.com](dh-lcgd.blogspot.com)

To use Blogger, I need to point the CNAME to ghs.google.com and the other CNAME is used for Google to authenticate that I am allowed to use this domain. The site is currently not accessible via the new domain is-a.dev until the CNAME is added and Google allows me to update the new domain name to Blogger

File: domains/kitb2romjbz3.dhlcgd.json (hunk 1 above)
Comment (dhlcgd) [reply domains/kitb2romjbz3.dhlcgd.json:7]: My website is live at [dh-lcgd.blogspot.com](dh-lcgd.blogspot.com)

To use Blogger, I need to point the CNAME to ghs.google.com and the other CNAME is used for Google to authenticate that I am allowed to use this domain. The site is currently not accessible via the new domain is-a.dev until the CNAME is added and Google allows me to update the new domain name to Blogger

**Comment by dhlcgd** [comment]
You can check my website at [dh-lcgd.blogspot.com](dh-lcgd.blogspot.com)

To use Blogger, I need to point the CNAME record to ghs.google.com and the other CNAME (gv-zz3nklni3izils.dv.googlehosted.com) is used for Google to authenticate that I am allowed to use this domain.

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**